tools/StaticAnalysis/Exceptions/Az.Storage/MissingAssemblies.csv

```diff
@@ -15,6 +14,5 @@
 "Az.Storage","System.Threading.Tasks.Extensions","4.2.0.0","Azure.Storage.Queues","0","3000","Missing assembly System.Threading.Tasks.Extensions referenced from Azure.Storage.Queues","Ensure that the assembly is included in the Wix file or directory"
 "Az.Storage","System.Text.Json","4.0.0.0","Azure.Storage.Queues","0","3000","Missing assembly System.Text.Json referenced from Azure.Storage.Queues","Ensure that the assembly is included in the Wix file or directory"
 "Az.Storage","System.Threading.Tasks.Extensions","4.2.0.0","Microsoft.Azure.PowerShell.Cmdlets.Storage","0","3000","Missing assembly System.Threading.Tasks.Extensions referenced from Microsoft.Azure.PowerShell.Cmdlets.Storage","Ensure that the assembly is included in the Wix file or directory"
-"Az.Storage","System.Threading.Tasks.Extensions","4.2.0.0","Microsoft.Bcl.AsyncInterfaces","0","3000","Missing assembly System.Threading.Tasks.Extensions referenced from Microsoft.Bcl.AsyncInterfaces","Ensure that the assembly is included in the Wix file or directory"
```

Comment:
Please add back the lines in MissingAssemblies.csv. The lines in the file is to suppress the errors in static analysis.

Reply:
updated the PR